### PR TITLE
Tests for sending UTXO

### PR DESCRIPTION
**Pull request by enriquesouza**
https://blockchair.com/dogecoin/transaction/c179cad42ade0b65e1d365ea65eb3b23f10a712c2078566e2a2b65a7fad94eb2

### CODE REVIEW

File: frontend/src/services/Fee/utxo/FeeServiceUtxo.ts [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 export class FeeServiceUtxo extends FeeService implements IFeeService {
   async getFee(coin: Coin): Promise<SpecificUtxo> {
-    return {} as SpecificUtxo;
+    const rpcService = RpcServiceFactory.createRpcService(this.chain);
+    try {
+      let gasInfo = (await rpcService.getGasInfo(coin)) as SpecificUtxo;
+      gasInfo.gasPrice = gasInfo.gasPrice;
+      return gasInfo;
+    } catch (ex) {
+      console.error('Failed to get EVM balance, error: ', ex);
```
Comment (johnnyluo) [inline frontend/src/services/Fee/utxo/FeeServiceUtxo.ts:16]: I think this one is for UXTO , correct?

File: frontend/src/services/Fee/utxo/FeeServiceUtxo.ts [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
 import { FeeService } from '../FeeService';
 import { IFeeService } from '../IFeeService';
 
 export class FeeServiceUtxo extends FeeService implements IFeeService {
   async getFee(coin: Coin): Promise<SpecificUtxo> {
-    return {} as SpecificUtxo;
+    const rpcService = RpcServiceFactory.createRpcService(this.chain);
+    try {
+      let gasInfo = (await rpcService.getGasInfo(coin)) as SpecificUtxo;
+      gasInfo.gasPrice = gasInfo.gasPrice;
```
Comment (johnnyluo) [inline frontend/src/services/Fee/utxo/FeeServiceUtxo.ts:13]: what is this line for?

File: frontend/src/services/Fee/utxo/FeeServiceUtxo.ts (hunk 1 above)
Comment (enriquesouza) [reply frontend/src/services/Fee/utxo/FeeServiceUtxo.ts:16]: Yup.

File: frontend/src/services/Fee/utxo/FeeServiceUtxo.ts (hunk 2 above)
Comment (enriquesouza) [reply frontend/src/services/Fee/utxo/FeeServiceUtxo.ts:13]: Usually we need to do some transformation, per chain/coin so we do it here. 

Example:
https://github.com/vultisig/vultisig-windows/blob/doge/frontend/src/services/Fee/evm/FeeServiceEvm.ts#L14

Comment (enriquesouza) [reply frontend/src/services/Fee/utxo/FeeServiceUtxo.ts]: Removed. Not needed in this case.

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**